packages/yoastseo/src/languageProcessing/helpers/sentence/SentenceTokenizer.js

```diff
@@ -391,6 +391,31 @@ export default class SentenceTokenizer {
 		return currentSentence.endsWith( lastAbbreviation );
 	}
 
+	/**
+	 * Checks whether the given tokens are a valid html tag pair.
+	 * Note that this method is not a full html tag validator. It should be replaced with a better solution once the html parser is implemented.
+	 * @param {string} firstToken An opening html tag.
+	 * @param {string} lastToken A closing html tag.
+	 * @returns {boolean} True if the tokens are a valid html tag pair. Otherwise, False.
+	 */
+	isValidTagPair( firstToken, lastToken ) {
+		const firstTokenText = firstToken.src;
+		const lastTokenText = lastToken.src;
+
+		// regex to get the tag type
+		const tagTypeRegex = /<\/?([^\s]+?)(\s|>)/;
+
+		// get the tag types
+		const firstTagType = firstTokenText.match( tagTypeRegex )[ 1 ];
+		const lastTagType  = lastTokenText.match( tagTypeRegex )[ 1 ];
+
+		// semantic tags (as opposed to style tags) are tags that are used to structure the text
+		const semanticTags = [ "p", "div", "h1", "h2", "h3", "h4", "h5", "h6", "span", "li", "main" ];
```

Comment:
* Question for CR: would it be worth it to make `semanticTags` this a global variable?
* Question for CR: Suggestions for a better name than `semanticTags` are welcome.

Reply:
Yes, same reasoning as the above comment. I think it's worth it to make it a global variable. And I think the naming is clear enough since you also add a comment above it :)

Reply:
And related to comments in general, I think it's more readable to start with a capital letter and end with a period. And inside `yoastseo` we use this style :)